pkg/webhook/pod/pd_deleter.go

```diff
@@ -207,9 +207,9 @@ func (pc *PodAdmissionControl) transferPDLeader(payload *admitPayload) *admissio
 
 	lastOrdinal := helper.GetMaxPodOrdinal(*payload.ownerStatefulSet.Spec.Replicas, payload.ownerStatefulSet)
 	if ordinal == lastOrdinal {
```

Comment:
This should use a similar logic with that in `pkg/manager/member/pd_upgrader.go`.

Reply:
add transfer leader to peer member logic